core/common/src/main/java/alluxio/Constants.java

```diff
@@ -152,6 +152,13 @@
   public static final String UNDERFS_S3_PROXY_HOST = "alluxio.underfs.s3.proxy.host";
   public static final String UNDERFS_S3_PROXY_PORT = "alluxio.underfs.s3.proxy.port";
   public static final String UNDERFS_S3_PROXY_HTTPS_ONLY = "alluxio.underfs.s3.proxy.https.only";
+  public static final String UNDERFS_S3_ENDPOINT = "alluxio.underfs.s3.endpoint";
```

Comment:
@ifcharming ,sure, I will.
The currently setting can only use "s3.amazonaws.com" as the service provider, if user try to use another one, then could not set it. Not all user use S3 APIs to connect to amazon service only, some other open source projects such as ceph, they provide the compatible s3 APIs. If user like to test with another AWS compatible service, they can set those S3 endpoints.